helpdesk_mgmt/readme/CONFIGURE.rst

```diff
@@ -74,3 +74,13 @@ Tags
 .. figure:: ../static/description/Tags.PNG
    :alt: Ticket Tags
    :width: 600 px
+
+
+Permissions
+~~~~~~~~~~~
+
+There are restrictions to read tickets according to the user's permissions set in Helpdesk.
+
+#. *User: Personal tickets*: User can list his tickets (those that are assigned to his user) or those that are not assigned to anyone.
```

Comment:
Use gender neutral words:

```suggestion
#. *User: Personal tickets*: User can list their tickets (those that are assigned to their user) or those that are not assigned to anyone.
```

Reply:
And this should be that the tickets are only visible if no team nor user is assigned.